server/src/api/listings.ts

```diff
@@ -29,7 +28,22 @@ listingRouter.get(
   '/',
   async (req: Request, res: Response, next: NextFunction) => {
     try {
-      const listings = await listingService.getAllListings();
+      const listing = await listingService.getAllListings();
```

Comment:
listings

Reply:
Oh I just realised this branch is outdated after I addressed my comments on the previous branch (which I merged)